README.md

```diff
@@ -3,6 +3,52 @@
 The members' data API is a Play app that manages and retrieves supporter attributes associated with a user.  
 It runs on https://members-data-api.theguardian.com/.
 
+
+## How do we handle Zuora 40 concurrent requests limit?
```

Comment:
actually on 16th april the API concurrency limits on your tenant have been doubled.  So we actually have 80 now!

Reply:
they were indeed doubled, but doubled from 20 back to 40
somewhere in the last few years they had been reduced to 20 without us knowing

Reply:
🤦 so they took away with one hand and gave back with the other...